### DISCUSSION TITLE
Use connection state of module to initialize custom variable?

### DISCUSSION THREAD

**Discussion by Jibun-no-Kage** [bitfocus/companion#2937] [General]
Use connection state of module to initialize custom variable?  I found that I can change button text, background color, even the background image based on connection state.  But I need to set a custom variable to a given value based on the connection state.  How would one do this?

**Comment by thedist** [comment]
Create a trigger. Set to trigger when the condition becomes true. Set the condition to the module condition. Set the action to whatever you want to happen, such as setting a variable.

For future reference, it may be better to use the listed support methods [https://bitfocus.io/companion/support](https://bitfocus.io/companion/support) of either the Facebook group, or Slack, rather than to create multiple discussions for simple 'how to' questions.

**Comment by Jibun-no-Kage** [comment]
@thedist Ah... I have had mixed results with FaceBook, and Slack seems to be slower, in my experience, then github.  Most of the projects I use, and even worked on or with others, are on github, so I tend to use github, but github focused/oriented.  That said, don't want to abuse it either.
